tests/util.rs

```diff
@@ -171,11 +171,19 @@ impl Dir {
     }
 
     /// Returns the path to the ripgrep executable.
-    pub fn bin(&self) -> PathBuf {
-        if cfg!(windows) {
+    pub fn bin(&self) -> process::Command {
+        let rg = if cfg!(windows) {
             self.root.join("../rg.exe")
         } else {
             self.root.join("../rg")
+        };
```

Comment:
You can use [`std::env::consts::EXE_SUFFIX`](https://doc.rust-lang.org/stable/std/env/consts/constant.EXE_SUFFIX.html) here instead of branching on Windows if you want to save a `cfg`!

Reply:
Nice catch, thanks! Fixed!